### ISSUE TITLE
Build error (Swift 3.0 Xcode 8.0 Beta 2)

### ISSUE THREAD

**Issue by iamjono** [crossroadlabs/Markdown#3]
Added to Package.swift, then on `swift build`:

```
Cloning https://github.com/crossroadlabs/CDiscount.git
HEAD is now at 207cbc3 added modulemap
Resolved version: 0.1.0
Compile Swift Module 'Markdown' (3 sources)
<module-includes>:1:9: note: in file included from <module-includes>:1:
#import "cdiscount.h"
        ^
/.../Packages/CDiscount-0.1.0/cdiscount.h:22:10: error: 'mkdio.h' file not found
#include <mkdio.h>
         ^
```

**Comment by dileping** [comment]
@iamjono Hi, thanks for the feedback!

Have you done prerequisites?

```
brew install discount
```

**Comment by iamjono** [comment]
Yep...

```
$ brew install discount
Warning: discount-2.2.0 already installed
```

**Comment by dileping** [comment]
@iamjono: could you, please, check if it works with Carthage on your system?

**Comment by iamjono** [comment]
I'm not using Carthage... I'm on Xcode 8.0 Beta 2, using SPM.
If I'm using SPM it should work without Carthage I would have presumed?

**Comment by dileping** [comment]
@iamjono: Yes, it should. I'm just asking if you could try it with Carthage to find out if the issue is your system specific.

**Comment by iamjono** [comment]
Yeah I did just try it with Carthage - no joy.

Note that there was already an entry in the linked frameworks and libraries for the Markdown.framework.

I should also note that it doesn't compile on the commandline using `swift build`, nor in Xcode.

**Comment by dileping** [comment]
@iamjono: thanks a lot! Looks like something new. Will take a look at it asap.

**Comment by iamjono** [comment]
That would be awesome, thanks!

**Comment by ypopovych** [comment]
Hi @iamjono ,
For the proper build, you have to add a header and library path to `swift build` command. So your command will be like this:
`swift build -Xcc -I/usr/local/include -Xlinker -L/usr/local/lib`

But seems as Markdown isn't ported to Swift 3.0 yet :(

**Comment by iamjono** [comment]
Yep figured as much, on both counts.

I really wanted to use it for generating the HTML docs from markdown for https://github.com/PerfectlySoft/PerfectDocs  :)

**Comment by dileping** [comment]
@iamjono @ypopovych I will get my hands on this this week to make it both Swift 2/3 compatible

**Comment by dileping** [comment]
@ypopovych @iamjono I've published a new version which supports Swift 3.0 (works with the newest Xcode). For now there is no possibility to build it without additional flags on Mac with `swift build` as there is no support for `pkg-config` in the underlying system library.

So, to build it on a Mac with `swift build` you still have to use these flags:

```
swift build -Xcc -I/usr/local/include -Xlinker -L/usr/local/lib
```

I will talk to the system library developers about the `pkg-config` issue. Let's see what we can do.

**Comment by dileping** [comment]
@iamjono @ypopovych Some good news here, guys. I've made a PR to `libmarkdown` with `pkg-config` and it's merged already. Waiting for a new release.
